config/initializers/pg_search.rb

```diff
@@ -1,7 +1,13 @@
 # See:
-# https://github.com/Casecommons/pg_search#trigram
+# https://github.com/Casecommons/pg_search#trigram-trigram-search
 # https://github.com/Casecommons/pg_search#ignoring-accent-marks
 PgSearch.multisearch_options = {
-  using: %i[tsearch trigram],
+  # using: %i[tsearch trigram],
+  using: {
+    tsearch: {
```

Comment:
I just added two basic options for now because I believe the match what we do with Elasticearch.